### ISSUE TITLE
Error in auto-label with HF Zephyr model

### ISSUE THREAD

**Issue by dhruv-anand-aintech** [enjalot/latent-scope#31]
Getting this error:

Running label
ls-label medium_articles_2 id cluster-001 transformers-HuggingFaceH4___zephyr-7b-beta ""

```
Loading environment variables from: /Users/dhruvanand/Code/latent-scope/latentscope-working/.env
RUNNING: cluster-001-labels-001

config.json:   0%|          | 0.00/608 [00:00<?, ?B/s]
config.json: 100%|██████████| 608/608 [00:00<00:00, 982kB/s]

model.safetensors:   0%|          | 0.00/2.20G [00:00<?, ?B/s]
model.safetensors:   0%|          | 10.5M/2.20G [00:00<02:01, 18.0MB/s]
model.safetensors:   1%|          | 21.0M/2.20G [00:01<01:42, 21.2MB/s]
model.safetensors:   1%|▏         | 31.5M/2.20G [00:01<01:47, 20.1MB/s]
model.safetensors:   2%|▏         | 41.9M/2.20G [00:02<01:49, 19.7MB/s]
model.safetensors:   2%|▏         | 52.4M/2.20G [00:02<01:50, 19.5MB/s]
model.safetensors:   3%|▎         | 62.9M/2.20G [00:03<01:50, 19.3MB/s]
model.safetensors:   3%|▎         | 73.4M/2.20G [00:03<01:50, 19.2MB/s]
model.safetensors:   4%|▍         | 83.9M/2.20G [00:04<01:50, 19.2MB/s]
model.safetensors:   4%|▍         | 94.4M/2.20G [00:04<01:50, 19.1MB/s]
model.safetensors:   5%|▍         | 105M/2.20G [00:05<01:49, 19.1MB/s]
model.safetensors:   5%|▌         | 115M/2.20G [00:05<01:49, 19.1MB/s]
model.safetensors:   6%|▌         | 126M/2.20G [00:06<01:49, 19.0MB/s]
model.safetensors:   6%|▌         | 136M/2.20G [00:07<01:48, 19.0MB/s]
model.safetensors:   7%|▋         | 147M/2.20G [00:07<01:47, 19.0MB/s]
model.safetensors:   7%|▋         | 157M/2.20G [00:08<01:47, 19.0MB/s]
model.safetensors:   8%|▊         | 168M/2.20G [00:08<01:47, 19.0MB/s]
model.safetensors:   8%|▊         | 178M/2.20G [00:09<01:46, 19.0MB/s]
model.safetensors:   9%|▊         | 189M/2.20G [00:10<01:57, 17.1MB/s]
model.safetensors:   9%|▉         | 199M/2.20G [00:10<01:47, 18.6MB/s]
model.safetensors:  10%|▉         | 210M/2.20G [00:11<01:46, 18.7MB/s]
model.safetensors:  10%|█         | 220M/2.20G [00:11<01:45, 18.8MB/s]
model.safetensors:  10%|█         | 231M/2.20G [00:12<01:44, 18.9MB/s]
model.safetensors:  11%|█         | 241M/2.20G [00:12<01:43, 18.9MB/s]
model.safetensors:  11%|█▏        | 252M/2.20G [00:13<01:42, 19.0MB/s]
model.safetensors:  12%|█▏        | 262M/2.20G [00:13<01:42, 19.0MB/s]
model.safetensors:  12%|█▏        | 273M/2.20G [00:14<01:41, 19.0MB/s]
model.safetensors:  13%|█▎        | 283M/2.20G [00:14<01:41, 19.0MB/s]
model.safetensors:  13%|█▎        | 294M/2.20G [00:15<01:40, 19.0MB/s]
model.safetensors:  14%|█▍        | 304M/2.20G [00:16<01:39, 19.0MB/s]
model.safetensors:  14%|█▍        | 315M/2.20G [00:16<01:39, 19.0MB/s]
model.safetensors:  15%|█▍        | 325M/2.20G [00:17<01:38, 19.0MB/s]
model.safetensors:  15%|█▌        | 336M/2.20G [00:17<01:37, 19.0MB/s]
model.safetensors:  16%|█▌        | 346M/2.20G [00:18<01:37, 19.0MB/s]
model.safetensors:  16%|█▌        | 357M/2.20G [00:18<01:37, 19.0MB/s]
model.safetensors:  17%|█▋        | 367M/2.20G [00:19<01:36, 19.0MB/s]
model.safetensors:  17%|█▋        | 377M/2.20G [00:19<01:35, 19.1MB/s]
model.safetensors:  18%|█▊        | 388M/2.20G [00:20<01:40, 18.1MB/s]
model.safetensors:  18%|█▊        | 398M/2.20G [00:21<01:38, 18.3MB/s]
model.safetensors:  19%|█▊        | 409M/2.20G [00:21<01:36, 18.5MB/s]
model.safetensors:  19%|█▉        | 419M/2.20G [00:22<01:35, 18.7MB/s]
model.safetensors:  20%|█▉        | 430M/2.20G [00:22<01:34, 18.7MB/s]
model.safetensors:  20%|██        | 440M/2.20G [00:23<01:33, 18.8MB/s]
model.safetensors:  20%|██        | 451M/2.20G [00:23<01:32, 18.8MB/s]
model.safetensors:  21%|██        | 461M/2.20G [00:24<01:32, 18.9MB/s]
model.safetensors:  21%|██▏       | 472M/2.20G [00:24<01:31, 19.0MB/s]
model.safetensors:  22%|██▏       | 482M/2.20G [00:25<01:30, 19.0MB/s]
model.safetensors:  22%|██▏       | 493M/2.20G [00:26<01:29, 19.0MB/s]
model.safetensors:  23%|██▎       | 503M/2.20G [00:26<01:29, 19.0MB/s]
model.safetensors:  23%|██▎       | 514M/2.20G [00:27<01:28, 19.0MB/s]
model.safetensors:  24%|██▍       | 524M/2.20G [00:27<01:28, 19.0MB/s]
model.safetensors:  24%|██▍       | 535M/2.20G [00:28<01:27, 19.0MB/s]
model.safetensors:  25%|██▍       | 545M/2.20G [00:28<01:27, 19.0MB/s]
model.safetensors:  25%|██▌       | 556M/2.20G [00:29<01:26, 19.0MB/s]
model.safetensors:  26%|██▌       | 566M/2.20G [00:29<01:25, 19.0MB/s]
model.safetensors:  26%|██▌       | 577M/2.20G [00:30<01:33, 17.4MB/s]
model.safetensors:  27%|██▋       | 587M/2.20G [00:31<01:27, 18.5MB/s]
model.safetensors:  27%|██▋       | 598M/2.20G [00:31<01:25, 18.7MB/s]
model.safetensors:  28%|██▊       | 608M/2.20G [00:32<01:24, 18.8MB/s]
model.safetensors:  28%|██▊       | 619M/2.20G [00:32<01:23, 18.9MB/s]
model.safetensors:  29%|██▊       | 629M/2.20G [00:33<01:23, 18.9MB/s]
model.safetensors:  29%|██▉       | 640M/2.20G [00:33<01:22, 18.9MB/s]
model.safetensors:  30%|██▉       | 650M/2.20G [00:34<01:21, 19.0MB/s]
model.safetensors:  30%|███       | 661M/2.20G [00:34<01:21, 19.0MB/s]
model.safetensors:  31%|███       | 671M/2.20G [00:35<01:20, 19.0MB/s]
model.safetensors:  31%|███       | 682M/2.20G [00:36<01:19, 19.0MB/s]
model.safetensors:  31%|███▏      | 692M/2.20G [00:36<01:19, 19.0MB/s]
model.safetensors:  32%|███▏      | 703M/2.20G [00:37<01:18, 19.0MB/s]
model.safetensors:  32%|███▏      | 713M/2.20G [00:37<01:18, 19.0MB/s]
model.safetensors:  33%|███▎      | 724M/2.20G [00:38<01:17, 19.0MB/s]
model.safetensors:  33%|███▎      | 734M/2.20G [00:38<01:17, 19.0MB/s]
model.safetensors:  34%|███▍      | 744M/2.20G [00:39<01:16, 19.0MB/s]
model.safetensors:  34%|███▍      | 755M/2.20G [00:39<01:16, 18.8MB/s]
model.safetensors:  35%|███▍      | 765M/2.20G [00:40<01:18, 18.4MB/s]
model.safetensors:  35%|███▌      | 776M/2.20G [00:41<01:20, 17.6MB/s]
model.safetensors:  36%|███▌      | 786M/2.20G [00:41<01:18, 18.0MB/s]
model.safetensors:  36%|███▌      | 797M/2.20G [00:42<01:16, 18.3MB/s]
model.safetensors:  37%|███▋      | 807M/2.20G [00:42<01:15, 18.5MB/s]
model.safetensors:  37%|███▋      | 818M/2.20G [00:43<01:13, 18.7MB/s]
model.safetensors:  38%|███▊      | 828M/2.20G [00:43<01:12, 18.8MB/s]
model.safetensors:  38%|███▊      | 839M/2.20G [00:44<01:12, 18.9MB/s]
model.safetensors:  39%|███▊      | 849M/2.20G [00:45<01:11, 18.9MB/s]
model.safetensors:  39%|███▉      | 860M/2.20G [00:45<01:10, 19.0MB/s]
model.safetensors:  40%|███▉      | 870M/2.20G [00:46<01:10, 19.0MB/s]
model.safetensors:  40%|████      | 881M/2.20G [00:46<01:09, 19.0MB/s]
model.safetensors:  41%|████      | 891M/2.20G [00:47<01:08, 19.0MB/s]
model.safetensors:  41%|████      | 902M/2.20G [00:47<01:08, 18.9MB/s]
model.safetensors:  41%|████▏     | 912M/2.20G [00:48<01:07, 19.1MB/s]
model.safetensors:  42%|████▏     | 923M/2.20G [00:48<01:07, 19.0MB/s]
model.safetensors:  42%|████▏     | 933M/2.20G [00:49<01:06, 19.0MB/s]
model.safetensors:  43%|████▎     | 944M/2.20G [00:50<01:08, 18.4MB/s]
model.safetensors:  43%|████▎     | 954M/2.20G [00:50<01:06, 18.7MB/s]
model.safetensors:  44%|████▍     | 965M/2.20G [00:51<01:11, 17.2MB/s]
model.safetensors:  44%|████▍     | 975M/2.20G [00:51<01:07, 18.2MB/s]
model.safetensors:  45%|████▍     | 986M/2.20G [00:52<01:05, 18.5MB/s]
model.safetensors:  45%|████▌     | 996M/2.20G [00:52<01:04, 18.6MB/s]
model.safetensors:  46%|████▌     | 1.01G/2.20G [00:53<01:03, 18.7MB/s]
model.safetensors:  46%|████▌     | 1.02G/2.20G [00:54<01:02, 18.8MB/s]
model.safetensors:  47%|████▋     | 1.03G/2.20G [00:54<01:02, 18.9MB/s]
model.safetensors:  47%|████▋     | 1.04G/2.20G [00:55<01:01, 18.8MB/s]
model.safetensors:  48%|████▊     | 1.05G/2.20G [00:55<01:00, 18.9MB/s]
model.safetensors:  48%|████▊     | 1.06G/2.20G [00:56<01:00, 18.9MB/s]
model.safetensors:  49%|████▊     | 1.07G/2.20G [00:56<00:59, 19.0MB/s]
model.safetensors:  49%|████▉     | 1.08G/2.20G [00:57<00:59, 19.0MB/s]
model.safetensors:  50%|████▉     | 1.09G/2.20G [00:57<00:58, 19.0MB/s]
model.safetensors:  50%|█████     | 1.10G/2.20G [00:58<00:57, 19.0MB/s]
model.safetensors:  51%|█████     | 1.11G/2.20G [00:58<00:57, 19.0MB/s]
model.safetensors:  51%|█████     | 1.12G/2.20G [00:59<00:56, 19.0MB/s]
model.safetensors:  51%|█████▏    | 1.13G/2.20G [01:00<00:56, 19.0MB/s]
model.safetensors:  52%|█████▏    | 1.14G/2.20G [01:00<00:55, 19.0MB/s]
model.safetensors:  52%|█████▏    | 1.15G/2.20G [01:01<01:01, 17.1MB/s]
model.safetensors:  53%|█████▎    | 1.16G/2.20G [01:01<00:55, 18.5MB/s]
model.safetensors:  53%|█████▎    | 1.17G/2.20G [01:02<00:54, 18.7MB/s]
model.safetensors:  54%|█████▍    | 1.18G/2.20G [01:02<00:54, 18.8MB/s]
model.safetensors:  54%|█████▍    | 1.20G/2.20G [01:03<00:53, 18.9MB/s]
model.safetensors:  55%|█████▍    | 1.21G/2.20G [01:04<00:52, 18.9MB/s]
model.safetensors:  55%|█████▌    | 1.22G/2.20G [01:04<00:51, 19.0MB/s]
model.safetensors:  56%|█████▌    | 1.23G/2.20G [01:05<00:51, 19.0MB/s]
model.safetensors:  56%|█████▌    | 1.24G/2.20G [01:05<00:50, 19.0MB/s]
model.safetensors:  57%|█████▋    | 1.25G/2.20G [01:06<00:50, 19.0MB/s]
model.safetensors:  57%|█████▋    | 1.26G/2.20G [01:06<00:49, 19.0MB/s]
model.safetensors:  58%|█████▊    | 1.27G/2.20G [01:07<00:48, 19.0MB/s]
model.safetensors:  58%|█████▊    | 1.28G/2.20G [01:07<00:48, 19.0MB/s]
model.safetensors:  59%|█████▊    | 1.29G/2.20G [01:08<00:47, 19.0MB/s]
model.safetensors:  59%|█████▉    | 1.30G/2.20G [01:09<00:47, 19.0MB/s]
model.safetensors:  60%|█████▉    | 1.31G/2.20G [01:09<00:46, 19.0MB/s]
model.safetensors:  60%|██████    | 1.32G/2.20G [01:10<00:46, 19.0MB/s]
model.safetensors:  61%|██████    | 1.33G/2.20G [01:10<00:45, 19.0MB/s]
model.safetensors:  61%|██████    | 1.34G/2.20G [01:11<00:45, 19.0MB/s]
model.safetensors:  61%|██████▏   | 1.35G/2.20G [01:11<00:48, 17.6MB/s]
model.safetensors:  62%|██████▏   | 1.36G/2.20G [01:12<00:45, 18.4MB/s]
model.safetensors:  62%|██████▏   | 1.37G/2.20G [01:12<00:44, 18.6MB/s]
model.safetensors:  63%|██████▎   | 1.38G/2.20G [01:13<00:43, 18.7MB/s]
model.safetensors:  63%|██████▎   | 1.39G/2.20G [01:14<00:42, 18.8MB/s]
model.safetensors:  64%|██████▍   | 1.41G/2.20G [01:14<00:42, 18.9MB/s]
model.safetensors:  64%|██████▍   | 1.42G/2.20G [01:15<00:41, 18.9MB/s]
model.safetensors:  65%|██████▍   | 1.43G/2.20G [01:15<00:40, 19.0MB/s]
model.safetensors:  65%|██████▌   | 1.44G/2.20G [01:16<00:40, 19.0MB/s]
model.safetensors:  66%|██████▌   | 1.45G/2.20G [01:16<00:39, 19.0MB/s]
model.safetensors:  66%|██████▌   | 1.46G/2.20G [01:17<00:39, 19.0MB/s]
model.safetensors:  67%|██████▋   | 1.47G/2.20G [01:17<00:38, 19.0MB/s]
model.safetensors:  67%|██████▋   | 1.48G/2.20G [01:18<00:37, 19.0MB/s]
model.safetensors:  68%|██████▊   | 1.49G/2.20G [01:19<00:37, 19.0MB/s]
model.safetensors:  68%|██████▊   | 1.50G/2.20G [01:19<00:37, 18.9MB/s]
model.safetensors:  69%|██████▊   | 1.51G/2.20G [01:20<00:36, 18.8MB/s]
model.safetensors:  69%|██████▉   | 1.52G/2.20G [01:20<00:36, 18.9MB/s]
model.safetensors:  70%|██████▉   | 1.53G/2.20G [01:21<00:35, 18.9MB/s]
model.safetensors:  70%|███████   | 1.54G/2.20G [01:21<00:34, 19.0MB/s]
model.safetensors:  71%|███████   | 1.55G/2.20G [01:22<00:36, 18.0MB/s]
model.safetensors:  71%|███████   | 1.56G/2.20G [01:23<00:34, 18.3MB/s]
model.safetensors:  71%|███████▏  | 1.57G/2.20G [01:23<00:33, 18.5MB/s]
model.safetensors:  72%|███████▏  | 1.58G/2.20G [01:24<00:33, 18.7MB/s]
model.safetensors:  72%|███████▏  | 1.59G/2.20G [01:24<00:32, 18.8MB/s]
model.safetensors:  73%|███████▎  | 1.60G/2.20G [01:25<00:31, 18.8MB/s]
model.safetensors:  73%|███████▎  | 1.61G/2.20G [01:25<00:30, 18.9MB/s]
model.safetensors:  74%|███████▍  | 1.63G/2.20G [01:26<00:30, 19.0MB/s]
model.safetensors:  74%|███████▍  | 1.64G/2.20G [01:26<00:29, 19.0MB/s]
model.safetensors:  75%|███████▍  | 1.65G/2.20G [01:27<00:29, 19.0MB/s]
model.safetensors:  75%|███████▌  | 1.66G/2.20G [01:27<00:28, 19.0MB/s]
model.safetensors:  76%|███████▌  | 1.67G/2.20G [01:28<00:28, 19.0MB/s]
model.safetensors:  76%|███████▋  | 1.68G/2.20G [01:29<00:27, 19.0MB/s]
model.safetensors:  77%|███████▋  | 1.69G/2.20G [01:29<00:26, 19.0MB/s]
model.safetensors:  77%|███████▋  | 1.70G/2.20G [01:30<00:26, 19.0MB/s]
model.safetensors:  78%|███████▊  | 1.71G/2.20G [01:30<00:25, 19.0MB/s]
model.safetensors:  78%|███████▊  | 1.72G/2.20G [01:31<00:25, 19.0MB/s]
model.safetensors:  79%|███████▊  | 1.73G/2.20G [01:31<00:24, 19.0MB/s]
model.safetensors:  79%|███████▉  | 1.74G/2.20G [01:32<00:25, 17.8MB/s]
model.safetensors:  80%|███████▉  | 1.75G/2.20G [01:33<00:24, 18.3MB/s]
model.safetensors:  80%|████████  | 1.76G/2.20G [01:33<00:23, 18.6MB/s]
model.safetensors:  81%|████████  | 1.77G/2.20G [01:34<00:22, 18.7MB/s]
model.safetensors:  81%|████████  | 1.78G/2.20G [01:34<00:22, 18.8MB/s]
model.safetensors:  81%|████████▏ | 1.79G/2.20G [01:35<00:21, 18.9MB/s]
model.safetensors:  82%|████████▏ | 1.80G/2.20G [01:35<00:20, 18.9MB/s]
model.safetensors:  82%|████████▏ | 1.81G/2.20G [01:36<00:20, 18.9MB/s]
model.safetensors:  83%|████████▎ | 1.82G/2.20G [01:36<00:19, 19.0MB/s]
model.safetensors:  83%|████████▎ | 1.84G/2.20G [01:37<00:19, 19.0MB/s]
model.safetensors:  84%|████████▍ | 1.85G/2.20G [01:38<00:18, 19.0MB/s]
model.safetensors:  84%|████████▍ | 1.86G/2.20G [01:38<00:18, 19.0MB/s]
model.safetensors:  85%|████████▍ | 1.87G/2.20G [01:39<00:17, 19.0MB/s]
model.safetensors:  85%|████████▌ | 1.88G/2.20G [01:39<00:17, 19.0MB/s]
model.safetensors:  86%|████████▌ | 1.89G/2.20G [01:40<00:16, 19.0MB/s]
model.safetensors:  86%|████████▋ | 1.90G/2.20G [01:40<00:15, 19.0MB/s]
model.safetensors:  87%|████████▋ | 1.91G/2.20G [01:41<00:15, 19.0MB/s]
model.safetensors:  87%|████████▋ | 1.92G/2.20G [01:41<00:14, 19.0MB/s]
model.safetensors:  88%|████████▊ | 1.93G/2.20G [01:42<00:15, 17.2MB/s]
model.safetensors:  88%|████████▊ | 1.94G/2.20G [01:43<00:14, 18.5MB/s]
model.safetensors:  89%|████████▊ | 1.95G/2.20G [01:43<00:13, 18.7MB/s]
model.safetensors:  89%|████████▉ | 1.96G/2.20G [01:44<00:12, 18.8MB/s]
model.safetensors:  90%|████████▉ | 1.97G/2.20G [01:44<00:12, 18.8MB/s]
model.safetensors:  90%|█████████ | 1.98G/2.20G [01:45<00:11, 18.5MB/s]
model.safetensors:  91%|█████████ | 1.99G/2.20G [01:45<00:11, 18.6MB/s]
model.safetensors:  91%|█████████ | 2.00G/2.20G [01:46<00:10, 18.7MB/s]
model.safetensors:  92%|█████████▏| 2.01G/2.20G [01:46<00:09, 18.8MB/s]
model.safetensors:  92%|█████████▏| 2.02G/2.20G [01:47<00:09, 18.9MB/s]
model.safetensors:  92%|█████████▏| 2.03G/2.20G [01:48<00:08, 18.9MB/s]
model.safetensors:  93%|█████████▎| 2.04G/2.20G [01:48<00:08, 18.9MB/s]
model.safetensors:  93%|█████████▎| 2.06G/2.20G [01:49<00:07, 19.0MB/s]
model.safetensors:  94%|█████████▍| 2.07G/2.20G [01:49<00:07, 19.0MB/s]
model.safetensors:  94%|█████████▍| 2.08G/2.20G [01:50<00:06, 19.0MB/s]
model.safetensors:  95%|█████████▍| 2.09G/2.20G [01:50<00:05, 19.0MB/s]
model.safetensors:  95%|█████████▌| 2.10G/2.20G [01:51<00:05, 19.0MB/s]
model.safetensors:  96%|█████████▌| 2.11G/2.20G [01:51<00:04, 19.0MB/s]
model.safetensors:  96%|█████████▋| 2.12G/2.20G [01:52<00:04, 19.0MB/s]
model.safetensors:  97%|█████████▋| 2.13G/2.20G [01:53<00:03, 18.0MB/s]
model.safetensors:  97%|█████████▋| 2.14G/2.20G [01:53<00:03, 18.3MB/s]
model.safetensors:  98%|█████████▊| 2.15G/2.20G [01:54<00:02, 18.5MB/s]
model.safetensors:  98%|█████████▊| 2.16G/2.20G [01:54<00:02, 18.7MB/s]
model.safetensors:  99%|█████████▊| 2.17G/2.20G [01:55<00:01, 18.8MB/s]
model.safetensors:  99%|█████████▉| 2.18G/2.20G [01:55<00:01, 18.8MB/s]
model.safetensors: 100%|█████████▉| 2.19G/2.20G [01:56<00:00, 18.9MB/s]
model.safetensors: 100%|██████████| 2.20G/2.20G [01:56<00:00, 18.9MB/s]
model.safetensors: 100%|██████████| 2.20G/2.20G [01:56<00:00, 18.8MB/s]

generation_config.json:   0%|          | 0.00/124 [00:00<?, ?B/s]
generation_config.json: 100%|██████████| 124/124 [00:00<00:00, 452kB/s]

tokenizer_config.json:   0%|          | 0.00/1.29k [00:00<?, ?B/s]
tokenizer_config.json: 100%|██████████| 1.29k/1.29k [00:00<00:00, 14.3MB/s]

tokenizer.model:   0%|          | 0.00/500k [00:00<?, ?B/s]
tokenizer.model: 100%|██████████| 500k/500k [00:00<00:00, 2.99MB/s]
tokenizer.model: 100%|██████████| 500k/500k [00:00<00:00, 2.99MB/s]

tokenizer.json:   0%|          | 0.00/1.84M [00:00<?, ?B/s]
tokenizer.json: 100%|██████████| 1.84M/1.84M [00:00<00:00, 2.28MB/s]
tokenizer.json: 100%|██████████| 1.84M/1.84M [00:00<00:00, 2.28MB/s]

special_tokens_map.json:   0%|          | 0.00/551 [00:00<?, ?B/s]
special_tokens_map.json: 100%|██████████| 551/551 [00:00<00:00, 6.18MB/s]
Traceback (most recent call last):
File "/Users/dhruvanand/miniforge3/bin/ls-label", line 8, in <module>
sys.exit(main())
File "/Users/dhruvanand/miniforge3/lib/python3.10/site-packages/latentscope/scripts/label_clusters.py", line 49, in main
labeler(args.dataset_id, args.text_column, args.cluster_id, args.model_id, args.context, args.rerun)
File "/Users/dhruvanand/miniforge3/lib/python3.10/site-packages/latentscope/scripts/label_clusters.py", line 108, in labeler
text = '\n'.join([f"{i+1}. {t}" for i, t in enumerate(items) if not too_many_duplicates(t)])
File "/Users/dhruvanand/miniforge3/lib/python3.10/site-packages/latentscope/scripts/label_clusters.py", line 108, in <listcomp>
text = '\n'.join([f"{i+1}. {t}" for i, t in enumerate(items) if not too_many_duplicates(t)])
File "/Users/dhruvanand/miniforge3/lib/python3.10/site-packages/latentscope/scripts/label_clusters.py", line 32, in too_many_duplicates
words = line.split()
AttributeError: 'int' object has no attribute 'split'
```

**Comment by enjalot** [comment]
this looks like you're labeling the `id` column, which might be numbers? I just pushed a commit to coerce the input to a string.
when ingesting or on the setup page you can set the text_column for the dataset which is used to extract the text for summarization (same as the embedding column)

**Comment by dhruv-anand-aintech** [comment]
thanks.

Another small issue came up when I used a text column which had a "|" in its name.

Running label
ls-label medium_articles_2 title|publication cluster-001 transformers-HuggingFaceH4___zephyr-7b-beta ""
```
/bin/sh: publication: command not found
usage: ls-label [-h] [--rerun RERUN]
dataset_id text_column cluster_id model_id context
ls-label: error: the following arguments are required: cluster_id, model_id, context
```

Of course, I could rather use some other separator.
It comes from my reembed script (https://github.com/AI-Northstar-Tech/vector-io/blob/b7e8e6823fa855a94f3a3790ecb76bb4f8a80eb0/src/vdf_io/scripts/reembed.py#L108), where I use it to indicate concatenation of different text columns before embedding.
Open to suggestions.

**Comment by enjalot** [comment]
hmm, i think its a good idea to quote the text_column when calling the scripts. I'll update that so its the default when a command is called by the ui

**Comment by enjalot** [comment]
I pushed fixes to both issues in this thread to 0.1.4
